CMakeLists.txt

```diff
@@ -4,8 +4,8 @@ project(student_proect)
 ## Compile as C++11, supported in ROS Kinetic and newer
 add_compile_options(-std=c++11)
 
-find_package(OpenCV REQUIRED )
-find_package(project_interface REQUIRED )
+##find_package(OpenCV REQUIRED )
```

Comment:
Maybe it's better to remove these comments. Otherwise we may forget to remove them when we start working on the actual VM.